src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Geometry/FormatCoordinates/FormatCoordinates.xaml.cs

```diff
@@ -69,21 +69,43 @@ private void InputTextChanged(object sender, TextChangedEventArgs e)
             _selectedTextField = (TextBox)sender;
         }
 
-        private void UpdateUIFromMapPoint(MapPoint startingPoint)
+        private void UpdateUIFromMapPoint(MapPoint selectedPoint)
         {
+            try
+            {
+                // Check if the selected point can be formatted into coordinates.
+                CoordinateFormatter.ToLatitudeLongitude(selectedPoint, LatitudeLongitudeFormat.DecimalDegrees, 0);
+            }
+            catch (Exception e)
```

Comment:
It looks like this is thrown as an `ArgumentException`. I think you could make the catch a bit more specific to just ArgumentException and then you wouldn't need an `if`. I'd also consider clearing out the textboxes and showing a mesage/alert/etc with the exception.